estate/models/estate.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+from dateutil.relativedelta import *
```

Comment:
Odoo guidelines would say we start with external libraries. Also, we prefer to name all imported libs, not using *
```suggestion
from dateutil.relativedelta import relativedelta

from odoo import fields, models
```